assets/scammer/scammer.json

```diff
@@ -62,6 +62,17 @@
             ],
             "submittedBy": "ohld",
             "submissionTimestamp": "2025-01-28T00:00:00Z"
+        },
+        {
+            "address": "EQD9FHnDxAOWM-8JEODWbHeW0XEPXDs3WpyEpja2ofJfqfsO",
```

Comment:
@ohld do we need separate file for this case?

Reply:
I'm not sure yet. Let's simply add a comment and keep this address in the same json file.